lib/cocoapods/installer/project_cache/target_cache_key.rb

```diff
@@ -162,8 +162,11 @@ def self.from_aggregate_target(sandbox, aggregate_target)
             'BUILD_SETTINGS_CHECKSUM' => build_settings,
           }
           if aggregate_target.includes_resources? || aggregate_target.includes_on_demand_resources?
-            relative_file_paths = aggregate_target.resource_paths_by_config.values.flatten.uniq + aggregate_target.on_demand_resources.map(&:to_s)
-            contents['FILES'] = relative_file_paths.sort_by(&:downcase)
+            relative_resource_file_paths = aggregate_target.resource_paths_by_config.values.flatten.uniq
+            relative_on_demand_resource_file_paths = aggregate_target.on_demand_resources.map do |res|
```

Comment:
this fixes an issue in which the target cache key for incremental installation was storing absolute paths for on demand resources. This would break incremental installation across developers but it is now fixed.